### DISCUSSION TITLE
Title been overwritten even existed in metadata

### DISCUSSION THREAD

**Discussion by wsyxbcl** [photoprism/photoprism#4102] [Q&A] [answered]
I encountered an issue with image titles while uploading images to the repository. Here's an example:

    Original filename: N7-测试.jpg
    Title in metadata: N7

After uploading the image, I noticed that the title in the repository became "测试" instead of "N7", even though the metadata had specified "N7" as the title. Additionally, when I attempted to change the filename to N7.jpg, the title of the image became "Unknown" after the update.

Is it possible that titles with certain patterns, such as "N7", "N70", being ignored by PhotoPrism when parsing the metadata. If so, what is this specific rules of ignoring (so I can avoid it), and is that the reasonable behavior (to overwrite titles even users specify them)?

**Comment by lastzero** [comment, marked as answer]
Yes, it seems possible that there is a sanity check that ignores very short titles consisting only of ASCII characters. For details, see our publicly available source code.

**Reply by wsyxbcl** [reply]
That makes sense, thanks for reply.
